components/brave_vpn/brave_vpn_service.cc

```diff
@@ -348,7 +343,13 @@ void BraveVpnService::Connect() {
 
   VLOG(2) << __func__ << " : start connecting!";
   UpdateAndNotifyConnectionStateChange(ConnectionState::CONNECTING);
-
+#if !BUILDFLAG(IS_ANDROID)
```

Comment:
I think here is already desktop only region?

Reply:
removed